### DISCUSSION TITLE
i am facing this issue : - Failed to resolve import "../backend/graphql" from "src/helpers/teams/BackendUserInfo.ts". Does the file exist?

### DISCUSSION THREAD

**Discussion by shubhamy17** [hoppscotch/hoppscotch#2867] [General]
![image](https://user-images.githubusercontent.com/65390183/205225930-b5888acc-6201-493a-b7fd-a01596fb6e80.png)

**Comment by AndrewBastin** [comment]
@shubhamy17 have you ran `pnpm install`, is it giving any errors ? Or, have you gotten the `.env` file configured properly ?
